lisa/sut_orchestrator/azure/common.py

```diff
@@ -442,6 +458,71 @@ def wait_copy_blob(
     log.debug("vhd copied")
 
 
+def connection_string(account: str, key: str):
```

Comment:
1. as our practice, use the f string
2. The method name should specify what the connection string is for.

Reply:
deleted this method